src/Models/Esp32Device.ts

```diff
@@ -75,6 +78,7 @@ export class Esp32Device extends ArduinoDeviceBase {
     if (templateFiles) {
       this.templateFiles = templateFiles;
     }
+    this.azureConfigFileHandler = new AzureConfigFileHandler(devicePath + "/..");

```

Comment:
Same as above and suggest have a variable such as "projectRootPath" to represent `devicePath + "/.."`, which is more humanreadable.

Reply:
Accept.